qemu/tests/cfg/multi_host.cfg

```diff
@@ -408,13 +408,16 @@
                     filesize = 100
                     transfer_timeout = 1000
 
-                - virtio_blk_data_plane:
+                - with_data_plane:
```

Comment:
@xutian 
Fair enough.
And code has been updated, could you please review it again?

Thanks.